### ISSUE TITLE
Pre-packaged working environment (e.g. Docker container)

### ISSUE THREAD

**Issue by rayshan** [joeferner/node-oracle#124]
Hi, thanks for all the hard work on this. Like many others, I struggled to get the module up and running on Ubuntu Server 13.10 64bit with Oracle 11g and node v0.10.x, both via npm and building from source.

Instead of supporting all the different environments separately, would it be more efficient to prepackage working environment in Docker containers and publish it on Docker index? That way contributors can focus on improvements.

FYI
http://www.docker.io/
"Docker ... easily create lightweight, portable, self-sufficient containers from any application. The same container that a developer builds and tests on a laptop can run at scale, in production, on VMs, bare metal, OpenStack clusters, public clouds and more."

Basically is an imaged diff of desired environment from the baseline stock ubuntu image. Very fast, small and portable.

Caveat is that this would be Linux-specific.
